tests/integration/test_quorum_inserts/test.py

```diff
@@ -161,78 +165,74 @@ def test_insert_quorum_with_drop_partition(started_cluster, add_new_data):
     print("Create Replicated table with three replicas")
     zero.query(create_query)
 
-    print("Stop fetches for test_quorum_insert_with_drop_partition at first replica.")
-    first.query("SYSTEM STOP FETCHES test_quorum_insert_with_drop_partition")
+    print(f"Stop fetches for {table_name} at first replica.")
+    first.query(f"SYSTEM STOP FETCHES {table_name}")
 
     print("Insert with quorum. (zero and second)")
-    zero.query(
-        "INSERT INTO test_quorum_insert_with_drop_partition(a,d) VALUES(1, '2011-01-01')"
-    )
+    zero.query(f"INSERT INTO {table_name}(a,d) VALUES(1, '2011-01-01')")
 
     print("Drop partition.")
-    zero.query(
-        "ALTER TABLE test_quorum_insert_with_drop_partition DROP PARTITION '2011-01-01'"
-    )
+    zero.query(f"ALTER TABLE {table_name} DROP PARTITION '2011-01-01'")
 
     if add_new_data:
         print("Insert to deleted partition")
-        zero.query(
-            "INSERT INTO test_quorum_insert_with_drop_partition(a,d) VALUES(2, '2011-01-01')"
-        )
+        zero.query(f"INSERT INTO {table_name}(a,d) VALUES(2, '2011-01-01')")
 
-    print("Resume fetches for test_quorum_insert_with_drop_partition at first replica.")
-    first.query("SYSTEM START FETCHES test_quorum_insert_with_drop_partition")
+    print(f"Resume fetches for {table_name} at first replica.")
+    first.query(f"SYSTEM START FETCHES {table_name}")
 
     print("Sync first replica with others.")
-    first.query("SYSTEM SYNC REPLICA test_quorum_insert_with_drop_partition")
+    first.query(f"SYSTEM SYNC REPLICA {table_name}")
 
     assert "20110101" not in first.query(
-        """
-    WITH (SELECT toString(uuid) FROM system.tables WHERE name = 'test_quorum_insert_with_drop_partition') AS uuid,
+        f"""
+    WITH (SELECT toString(uuid) FROM system.tables WHERE name = '{table_name}') AS uuid,
          '/clickhouse/tables/' || uuid || '/0/quorum/last_part' AS p
     SELECT * FROM system.zookeeper WHERE path = p FORMAT Vertical
     """
     )
 
+    # Sync second replica not to have `REPLICA_IS_NOT_IN_QUORUM` error
+    second.query(f"SYSTEM SYNC REPLICA {table_name}")
```

Comment:
Only actual change in the test (and same for `test_insert_quorum_with_move_partition`). Others is making table name unique for each test parameter value